### PR TITLE
Add screenshots to English documentation

### PR DESCRIPTION
**Pull request by jhoule86**
- common images
- english images
- add images to english documentation

**Reminder:** use the `view file` option to see the markdown and images rendered to HTML

### CODE REVIEW

File: meta/documents/user_guide/en/initial_setup.md [hunk 1]
```diff
@@ -71,11 +87,17 @@ This procedure is required only if [the `Send all inventory items to Wayfair` se
...
+
+        ![variation active](../../../images/en/variation_active_field.png)
+
+    2. If [the `Send all inventory items to Wayfair?` setting](settings_guide.md#send-all-inventory-items-to-wayfair) is **disabled**, go to the `Availability` tab of the Variation and add "Wayfair" to the list in the `Markets` area:
 
-    2. If [the `Send all inventory items to Wayfair?` setting](settings_guide.md#send-all-inventory-items-to-wayfair) is **disabled**, go to the `Availability` tab of the Variation and add "Wayfair" to the list in the `Markets` area.
+        ![adding wayfair market to variation](../../../images/en/variation_wayfair_market.png)
 
-    3. Click the `Save` button next to the Variation `ID` (not the higher-up button for the Item).
+    3. Click the `Save` next to the Variation `ID` (not the higher-up button for the Item):
```
Comment (rickyklu) [inline meta/documents/user_guide/en/initial_setup.md:98]:
```suggestion
    3. Click the `Save` to the left of the Variations `ID`, below `Global` (not the higher-up button for the Item):
```

Comment (rickyklu) [inline meta/documents/user_guide/en/initial_setup.md:98]:
```suggestion
    3. Click the `Save` left of the Variation `ID`, below the `Global` tab (not the higher-up button for the Item):

```

File: meta/documents/user_guide/en/initial_setup.md [hunk 2]
```diff
@@ -91,34 +113,59 @@ If Wayfair's shipping services are to be used, the Wayfair plugin's ASN settings
...
+
+    ![order events](../../../images/en/menu_order_events.png)
+
+2.	Click on `Add event procedure` (the `+` button on the bottom left-hand side of the page):
+
+    ![add order event](../../../images/en/add_order_event.png)
+
+3.	Enter any `Name` in the appropriate field.
+
+4.	Select `Status change` (in the category `Order Change`) in the `Event` field:
```
Comment (rickyklu) [inline meta/documents/user_guide/en/initial_setup.md:126]:
```suggestion
4.	In the `Event`  drop down, select `Status change` (below the category `Order Change`):
```

File: meta/documents/user_guide/en/initial_setup.md [hunk 3]
```diff
@@ -91,34 +113,59 @@ If Wayfair's shipping services are to be used, the Wayfair plugin's ASN settings
...
+
+    ![choose event](../../../images/en/shipping/choose_event.png)
+
+5.	In the field below `Event` select the status change that should initiate the sending of an ASN to Wayfair, such as `In preparation for shipping`:
+
+    ![choose status](../../../images/en/shipping/choose_status.png)
+
+6.	Click the `Save` button ![save button](../../../images/common/button_save.png).
+
+7.	You should automatically be redirected to the newly created event procedure. In the `settings` section of the event procedure, place a checkmark next to `Active`:
```
Comment (rickyklu) [inline meta/documents/user_guide/en/initial_setup.md:136]:
```suggestion
7.	You should automatically be redirected to the newly created event procedure. In the `Settings` section of the event procedure, place a checkmark next to `Active`:

```

File: meta/documents/user_guide/en/initial_setup.md [hunk 4]
```diff
@@ -91,34 +113,59 @@ If Wayfair's shipping services are to be used, the Wayfair plugin's ASN settings
...
+
+    ![event active](../../../images/en/shipping/event_active.png)
+
+8. Click on `Add Filter`:
```
Comment (rickyklu) [inline meta/documents/user_guide/en/initial_setup.md:140]:
```suggestion
8. Click on the `+` symbol next to `Filter`:

```

File: meta/documents/user_guide/en/initial_setup.md [hunk 5]
```diff
@@ -91,34 +113,59 @@ If Wayfair's shipping services are to be used, the Wayfair plugin's ASN settings
...
+10. Click the `Add` button ![plus add button](../../../images/en/button_plus_add.png).
 
-3.	Enter any `Name`
+11.	In the `Filter` section, a box should appear with a list of all available Order referrers. Place a checkmark next to all "Wayfair" order referrers:
 
-4.	Select `Status change` (in the category `Order Change`) in the `Event` field.
+    ![wayfair selected](../../../images/en/shipping/filter_wayfair_selected.png)
 
-5.	In the field below `Event` select the status change that should initiate the sending of an ASN to Wayfair, such as `in preparation for shipping`.
+12. Click on `Add procedure`:
```
Comment (rickyklu) [inline meta/documents/user_guide/en/initial_setup.md:154]:
```suggestion
12. Click on the `+` next to `Procedures`:

```

File: meta/documents/user_guide/en/initial_setup.md [hunk 6]
```diff
@@ -21,41 +21,57 @@
...
-5. In the `Supplier Settings` area, enter the `Client ID` and `Client Secret` values that correspond with your Wayfair API credentials
+    ![global settings in menu](../../../images/en/menu_global_settings.png)
 
-6. Change the `Mode` setting to `Live` - see [information on `Test` mode](test_mode.md)
+5. In the `Supplier Settings` area, enter the `Client ID` and `Client Secret` values that correspond with your Wayfair API credentials.
 
-7. Click the `Save` button in the toolbar above the settings
+6. Change the `Mode` setting to `Live` - see [information on `Test` mode](test_mode.md):
+
+    ![global settings live](../../../images/en/global_settings_live.png)
```
Comment (rickyklu) [inline meta/documents/user_guide/en/initial_setup.md:50]: Highlight the the "Mode *" dropdown so it is 100% clear where they have to change the Mode to `Live`

File: meta/documents/user_guide/en/initial_setup.md [hunk 7]
```diff
@@ -21,41 +21,57 @@
...
+
+7. Click the `Save` button ![save button](../../../images/common/button_save.png) in the toolbar above the settings
 
 ## 2. Activating the order referrer
 An order referrer in Plentymarkets identifies the sales channel on which an order was generated. To get the Plentymarkets system to properly import orders from the Wayfair API, the Wayfair order referrer must be activated:
 
-1. From the main Plentymarkets page, go to `Setup` >> `Orders` >> `Order referrer`.
+1. From the main Plentymarkets page, go to `Setup` >> `Orders` >> `Order referrer`:
+
+    ![order referrer menu](../../../images/en/menu_order_referrer.png)
```
Comment (rickyklu) [inline meta/documents/user_guide/en/initial_setup.md:59]: In the screenshot, maybe highlight the `Setup` tab, the `Orders` left side menu, and the `Order referrer` sub-menu? Or maybe add numbering to them?

File: meta/documents/user_guide/en/updating.md [hunk 8]
```diff
@@ -6,37 +6,61 @@ The Wayfair plugin may be updated at any time to ensure that it is providing the
 Prior to updating, review [the latest information on the Wayfair plugin's releases.](https://github.com/wayfair-contribs/plentymarkets-plugin/releases)
 
 ## Updating to the latest version of the Wayfair Plugin
-1. From the main Plentymarkets page, go to `Plugins` >> `Plugin set overview`
+1. From the main Plentymarkets page, go to `Plugins` >> `Plugin set overview`:
+
+    ![plugins_menu_plugin_set_overview](../../../images/en/plugins_menu_plugin_set_overview.png)
+
     * **Do not select the deprecated, similarly named `Plugin overview` option, if it appears in the menu.**
```
Comment (rickyklu) [inline meta/documents/user_guide/en/updating.md:13]: I would move this line up, below line 9. Better to instruct people what NOT to do beforehand if they are following this guide step-by-step

File: meta/documents/user_guide/en/updating.md [hunk 9]
```diff
@@ -6,37 +6,61 @@ The Wayfair plugin may be updated at any time to ensure that it is providing the
...
 
 7. Review the information on the page.
 
 9. In the `Select version` drop-down menu, choose [the newest plugin version](https://github.com/wayfair-contribs/plentymarkets-plugin/releases) if it is not already selected.
 
-10. Click the `Update` button at the top of the page. After a short pause, you will be redirected to the individual Plugin Set's details.
+    ![update selector](../../../images/en/installation/update_version_selection.png)
 
-11. Verify that a row exists on the page with the `Name` displayed as "Wayfair" and the `Installed` version showing as the desired version
+10. Click the `Update` button ![update button](../../../images/en/updating/update_button.png) at the top of the page. After a short pause, you will be redirected to the individual Plugin Set's details.
```
Comment (rickyklu) [inline meta/documents/user_guide/en/updating.md:46]: The image has a broken link when I was looking at the markdown.

Comment (jhoule86) [reply meta/documents/user_guide/en/updating.md:46]: thanks. yeah I forgot to refactor that link. There's no `updating` image directory anymore. It's all in `installation` now.

File: meta/documents/user_guide/en/initial_setup.md (hunk 7 above)
Comment (jhoule86) [reply meta/documents/user_guide/en/initial_setup.md:59]: When you use the Plentymarkets UI, there's progressive disclosure in place, so each option is only visible after choosing the one before it. I think numbering could be overkill, but I'll see if there are ways to highlight where to click.

Comment (jhoule86) [reply meta/documents/user_guide/en/initial_setup.md:59]: Discussed this via slack, closing.

File: meta/documents/user_guide/en/plugin_installation.md [hunk 10]
```diff
@@ -7,11 +7,13 @@ Follow these steps to make it available for use:
 
 1. Log in to the Plentymarkets system.
 
-2. From the main Plentymarkets page, go to `plugins` >> `plentyMarketplace`, which will use the Plentymarkets system's owner’s credentials to log you in on the plentyMarketplace website.
+2. From the main Plentymarkets page, go to `plugins` >> `plentyMarketplace`, which will use the Plentymarkets system's owner’s credentials to log you in on the plentyMarketplace website:
```
Comment (rickyklu) [inline meta/documents/user_guide/en/plugin_installation.md:10]:
```suggestion
2. From the main Plentymarkets page, go to `Plugins` >> `plentyMarketplace`
```

File: meta/documents/user_guide/en/plugin_installation.md [hunk 11]
```diff
@@ -27,45 +29,64 @@ Follow these steps to make it available for use:
...
 
-10. Click the `Install` button at the top of the page. You will be redirected to the individual Plugin Set's details.
+10. Click the `Install` button ![install button](../../../images/en/installation/button_install.png) at the top of the page. You will be redirected to the individual Plugin Set's details.
 
 11. Verify that a row exists on the page with the `Name` displayed as "Wayfair" and the `Installed` version showing as the desired version
 
 12. In the `Active` column for the "Wayfair" row, click on the switch so that it moves to the right-side, enabled position. **Failure to activate the Wayfair plugin will prevent its functionalities from being available**.
 
-13. Click the `Deploy plugin set` button (appears as a Save button) at the top of the page. A progress bar will appear while the plugin is installed.
+13. Verify that the Wayfair plugin's row looks like this:
```
Comment (rickyklu) [inline meta/documents/user_guide/en/plugin_installation.md:79]:
```suggestion
13. Verify that the Wayfair plugin's row looks the same as the screenshot below (your `Installed` version may be different)
```

File: meta/documents/user_guide/en/plugin_installation.md [hunk 12]
```diff
@@ -27,45 +29,64 @@ Follow these steps to make it available for use:
...
+
+    ![wayfair plugin not yet deployed](../../../images/en/installation/wayfair_plugin_not_yet_deployed.png)
+
+14. Click the `Deploy plugin set` button (appears as a Save button):
+
+    ![deploy plugin set button](../../../images/en/installation/button_deploy_plugin_set.png)
+
+15. A progress bar will appear while the plugin is installed.
+
+16. Confirm that the `Deployed` column for the "Wayfair" row is now populated, and it reflects the value in the `Installed` column:
```
Comment (rickyklu) [inline meta/documents/user_guide/en/plugin_installation.md:89]:
```suggestion
16. Confirm that the `Deployed` column for the "Wayfair" row is now populated, and it reflects the value in the `Installed` column (your `Installed` or `Deployed` version may be different)
```

File: meta/documents/user_guide/en/settings_guide.md [hunk 13]
```diff
@@ -24,20 +36,22 @@ The topography of the Plentymarkets system may not match the amount of Wayfair S
 When choosing a relationship between Warehouses and Supplier IDs that is not one-to-one, make sure to review the behavior of the [Stock Buffer](#stock-buffer) setting.
 
 ### Adding a Warehouse mapping
-1. Click on the `Add Mapping` button
+1. Click on the `Add Mapping` button ![add mapping](../../../images/en/wayfair_settings/warehouses/button_add_mapping.png).
```
Comment (rickyklu) [inline meta/documents/user_guide/en/settings_guide.md:39]: broken image link

**Pull request merged**